xml/2018-10-07.xml

```diff
@@ -0,0 +1,238 @@
+      chance to ask any questions you may have at the end of the talk.
+    </text>
+    <links>
+      <link href="https://www.facebook.com/events/1914035272235179/">Facebook</link>
+    </links>
+  </event>
+
+  <heading>Sponsors' Exhibition</heading>
+  <image src="https://i.imgur.com/ptAlUIl.jpg" width="100" />
+  <event startDate="2018-10-09T17:00:00+01:00"
+    endDate="2018-10-09T20:00:00+01:00"
+    location="Queens Tower Rooms">
+    <text>
+      Come to the Queens Tower Rooms on Tuesday 9th October between 5pm - 8pm to
+      have a chance to meet all of DoCSoc's Silver and Gold sponsors this year
+      and grab some swag.
+
+      This year's exciting sponsors that will be attending include Facebook,
+      Palantir, Bloomberg, Samsung, Netcraft, Microsoft, Next Jump, Entrepreneur
+      First, NewVoiceMedia and Artificial Labs.
+
+      Whether you are looking for an internship or graduate role, or simply want
+      to know more about industry, we really recommend you come along as it's a
+      great way to get in direct contact with recruiters and engineers from highly
+      regarded companies and should provide invaluable insight into the recruitment
+      process.
+
+      This event is open to all DoCSoc members (automatically includes Computing,
+      JMC, and EIE students).
+    </text>
+    <links>
+      <link href="https://www.facebook.com/events/1873849222722225/">Facebook</link>
+    </links>
+  </event>
+
+  <heading>Bloomberg Extreme Decentralised App Development</heading>
+  <event startDate="2018-10-11T18:00:00+01:00"
+    endDate="2018-10-11T21:00:00+01:00"
+    location="Skempton 060">
+    <text>
+      Join Bloomberg on Thursday 11th October at 6pm for a session of Extreme
+      Decentralised Application Development on the Ethereum blockchain. We will
+      present to you the tools you need to develop, test and deploy smart contracts
+      on the Ethereum blockchain. Then you will be given the chance to work in
+      groups to develop and deploy a series of smart contracts that will allow you
+      to create a toy social network on Ethereum.
+
+      We hope that you will leave the session knowing where to start with
+      decentralised app development and with a basic understanding of the issues
+      such apps face and why they haven't been widely adopted yet.
+
+      Note that if you're a first year and are new to programming, you may find
+      this event a little intense.
+
+      Drinks at the Union to follow.
+    </text>
+    <links>
+      <link href="Facebook">RSVP (Facebook)</link>
+    </links>
+  </event>
+
+  <hr />
+
+  <text>Next, here's a couple of events coming up in the longer term:</text>
+
+  <heading>CGCU Annual Welcome Dinner</heading>
+  <image src="https://i.imgur.com/LXIr2lJ.jpg" width="100" />
+  <event startDate="2018-10-27T18:30:00+01:00"
+    endDate="2018-10-28T01:00:00+01:00"
+    location="Millennium Gloucester Hotel">
```

Comment:
set `external="true"` here as well as DoCSoc isn't running this event.
`external` adds a disclaimer that it's an external event. Default value is false.